perllib/FixMyStreet/Cobrand/Kingston.pm

```diff
@@ -101,6 +110,293 @@ sub garden_waste_renewal_sacks_cost_pa {
+            }, {
+                value => CONTAINER_REFUSE_360,
+                label => 'Larger black rubbish bin',
+                disabled => $value->{disabled},
+                hint => 'You already have the biggest sized bin allowed. If you have an exceptionally large household or your household has medical needs that create more waste than normal, you can apply for more capacity, but this will be assessed by our officers.',
+            },
+            );
+        } elsif ($id == CONTAINER_REFUSE_360) {
+            $c->stash->{current_refuse_bin} = 360;
+            @radio_options = ( {
+                value => CONTAINER_REFUSE_180,
+                label => '180L black rubbish bin ‘standard’',
+                disabled => $value->{disabled},
+            }, {
+                value => CONTAINER_REFUSE_240,
+                label => '240L black rubbish bin ‘larger’',
+                disabled => $value->{disabled},
+            },
+            );
+        }
+    }
+
+    @$field_list = (
+        "container-capacity-change" => {
+            type => 'Select',
+            widget => 'RadioGroup',
+            label => 'Which container do you need?',
+            options => \@radio_options,
+            required => 1,
+        }
+    );
+}
+
+=head2 waste_request_form_first_next
+
+After picking a container, we ask what bins needs removing.
+
+=cut
+
+sub waste_request_form_first_title {
+    my $self = shift;
+    my $c = $self->{c};
+    return 'Black bin size change request' if $c->get_param('exchange');
+}
+
+sub waste_request_form_first_next {
+    my $self = shift;
+    my $c = $self->{c};
+    if ($c->get_param('exchange')) {
+        return sub {
+            my $data = shift;
+            my $choice = $data->{"container-capacity-change"};
+            if ($choice == CONTAINER_REFUSE_360) {
+                $c->res->redirect('https://www.kingston.gov.uk/'); # TODO
+                $c->detach;
+            } else {
+                $data->{"container-$choice"} = 1;
+                $data->{"quantity-$choice"} = 1;
+                $data->{"removal-$choice"} = 1;
+            }
+            return 'about_you';
+        };
+    }
+    return 'removals';
+}
+
+=head2 waste_munge_request_form_pages
+
+We have a separate removal page, asking which bins need to be removed.
+
+=cut
+
+sub waste_munge_request_form_pages {
+    my ($self, $page_list, $field_list) = @_;
+    my $c = $self->{c};
+
+    if (($c->stash->{current_refuse_bin} || 0) == 180) {
+        $c->stash->{first_page} = 'how_many_exchange';
+    }
+
+    my %maxes;
+    foreach (@{$c->stash->{service_data}}) {
+        next unless $_->{next} || $_->{request_only};
+        my $containers = $_->{request_containers};
+        my $maximum = $_->{request_max};
+        foreach my $id (@$containers) {
+            $maxes{$id} = $maximum->{$id};
+        }
+    }
+
+    sub n { my $n = ucfirst NUMWORDS(shift); $n =~ s/Zero/None/; $n; }
```

Comment:
I felt like I had to stop and think about my answer when I saw this:

<img width="384" alt="image" src="https://github.com/mysociety/fixmystreet/assets/4776/01fa72db-d3db-4af6-b971-42505d19ca8f">

Something about the numerals not being shown caught me by surprise. I think this would be a bit more accessible, maybe:

<img width="419" alt="image" src="https://github.com/mysociety/fixmystreet/assets/4776/922653e7-24b7-4113-a075-6bec35572e49">